packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart

```diff
@@ -15,209 +16,418 @@ import 'package:flutter_tools/src/base/user_messages.dart';
 import 'package:flutter_tools/src/cache.dart';
 
 import 'package:flutter_tools/src/project.dart';
+import 'package:standard_message_codec/standard_message_codec.dart';
 
 import '../src/common.dart';
 
 void main() {
 
-  Future<Map<String, List<String>>> extractAssetManifestFromBundle(ManifestAssetBundle bundle) async {
-    final String manifestJson = utf8.decode(await bundle.entries['AssetManifest.json']!.contentsAsBytes());
-    final Map<String, dynamic> parsedJson = json.decode(manifestJson) as Map<String, dynamic>;
-    final Iterable<String> keys = parsedJson.keys;
-    final Map<String, List<String>> parsedManifest = <String, List<String>> {
-      for (final String key in keys) key: List<String>.from(parsedJson[key] as List<dynamic>),
-    };
-    return parsedManifest;
-  }
-
-  group('AssetBundle asset variants (with Unix-style paths)', () {
-    late Platform platform;
-    late FileSystem fs;
-
-    setUp(() {
-      platform = FakePlatform();
-      fs = MemoryFileSystem.test();
-      Cache.flutterRoot = Cache.defaultFlutterRoot(
-        platform: platform,
-        fileSystem: fs,
-        userMessages: UserMessages()
-      );
-
-      fs.file('.packages').createSync();
-
-      fs.file('pubspec.yaml').writeAsStringSync(
-'''
-name: test
-dependencies:
+  // TODO(andrewkolos): Delete this group once we stop producing AssetManifest.json
```

Comment:
For TODOs with a task associated like this, Its good practice to file a cleanup bug and link it here

Reply:
Good idea. I'll get on it

Reply:
Added a link to https://github.com/flutter/flutter/issues/114913 in the TODO